.github/workflows/ui-ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```

Comment:
Nice :D I think it's fine to stick to one LTS version for ui, but I like keeping it to 14+16 (probably always LTS + last LTS) for API since that's a little more closely integrated with node.